WORKSPACE

```diff
@@ -0,0 +1,150 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
```

Comment:
Can you also update `README.md` about these gazelle instructions, and update `cloudbuild.yaml` to add a step to `bazel test` everything?

Reply:
Added bazel test to cloudbuild and updated README.